charts/studio/templates/configmap-studio-backend.yaml

```diff
@@ -7,3 +7,9 @@ data:
 {{-  toYaml . | nindent 2 }}
 {{- end }}
 {{- include "studio.dvcx.configMap" . | indent 2}}
+
+  LEO_SERVICE_HOST: "{{ .Release.Name }}-leo"
+  LEO_SERVICE_PORT: "{{ .Values.studioLeo.service.port }}"
+  LEO_ENCODING: "utf-8"
+  LEO_API_PATH_PREFIX: "{{- if and .Values.global.basePath (not (eq .Values.global.basePath "/")) }}{{ include "studio.basePath" . }}/{{- end }}leo"
```

Comment:
@dacbd Kindly ask you to check if the configuration is valid 😃 especially `LEO_API_PATH_PREFIX` - I configure it for support custom `basePath`

Reply:
Looks right to me, but the question is, where do we set the base path for the LEO server itself? cc @dacbd

Reply:
Actually, I'm not even sure that LEO needs to worry about base path since it's only running within the cluster, right?

Reply:
@jesper7 is correct, the leo api is internal only, so while the API was built to be able to have some prefix we don't need to set anything. I would leave it empty for leo to use its default.